[networkupstools] tests fail on windows

```
[INFO] -------------------------------------------------------
[INFO]  T E S T S
[INFO] -------------------------------------------------------
[INFO] Running org.openhab.binding.networkupstools.internal.nut.NutApiTest
[INFO] Tests run: 2, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.659 s - in org.openhab.binding.networkupstools.internal.nut.NutApiTest
[INFO] Running org.openhab.binding.networkupstools.internal.NutNameChannelsTest
[ERROR] Tests run: 2, Failures: 0, Errors: 2, Skipped: 0, Time elapsed: 0.015 s <<< FAILURE! - in org.openhab.binding.networkupstools.internal.NutNameChannelsTest
[ERROR] org.openhab.binding.networkupstools.internal.NutNameChannelsTest.testReadmeMatchingChannels  Time elapsed: 0 s  <<< ERROR!
java.nio.file.InvalidPathException: Illegal char <:> at index 2: /C:/Users/Jan/Documents/Git/openhab-addons/bundles/org.openhab.binding.networkupstools/target/test-classes/../..\README.md
        at org.openhab.binding.networkupstools.internal.NutNameChannelsTest.readReadme(NutNameChannelsTest.java:124)
        at org.openhab.binding.networkupstools.internal.NutNameChannelsTest.testReadmeMatchingChannels(NutNameChannelsTest.java:73)

[ERROR] org.openhab.binding.networkupstools.internal.NutNameChannelsTest.testNutNameMatchingReadme  Time elapsed: 0.015 s  <<< ERROR!
java.nio.file.InvalidPathException: Illegal char <:> at index 2: /C:/Users/Jan/Documents/Git/openhab-addons/bundles/org.openhab.binding.networkupstools/target/test-classes/../..\README.md
        at org.openhab.binding.networkupstools.internal.NutNameChannelsTest.readReadme(NutNameChannelsTest.java:124)
        at org.openhab.binding.networkupstools.internal.NutNameChannelsTest.testNutNameMatchingReadme(NutNameChannelsTest.java:104)

[INFO] Running org.openhab.binding.networkupstools.internal.NutNameTest
[INFO] Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.079 s - in org.openhab.binding.networkupstools.internal.NutNameTest
[INFO]
[INFO] Results:
[INFO]
[ERROR] Errors:
[ERROR]   NutNameChannelsTest.testNutNameMatchingReadme:104->readReadme:124 » InvalidPath
[ERROR]   NutNameChannelsTest.testReadmeMatchingChannels:73->readReadme:124 » InvalidPath
[INFO]
[ERROR] Tests run: 5, Failures: 0, Errors: 2, Skipped: 0
```

--- a/bundles/org.openhab.binding.networkupstools/src/test/java/org/openhab/binding/networkupstools/internal/NutNameChannelsTest.java
+++ b/bundles/org.openhab.binding.networkupstools/src/test/java/org/openhab/binding/networkupstools/internal/NutNameChannelsTest.java
@@ -17,8 +17,9 @@ import static org.hamcrest.Matchers.is;
 import static org.junit.jupiter.api.Assertions.*;
 
 import java.io.IOException;
+import java.net.URISyntaxException;
 import java.nio.file.Files;
-import java.nio.file.Paths;
+import java.nio.file.Path;
 import java.util.ArrayList;
 import java.util.List;
 import java.util.Map;
@@ -118,28 +119,29 @@ public class NutNameChannelsTest {
     }
 
     private Map<NutName, String> readReadme() {
-        final String path = getClass().getProtectionDomain().getClassLoader().getResource(".").getFile() + "../..";
-
         try {
-            final List<String> lines = Files.readAllLines(Paths.get(path, "README.md"));
+            final String path = Path.of(getClass().getProtectionDomain().getClassLoader().getResource(".").toURI())
+                    .toString();
+            final List<String> lines = Files.readAllLines(Path.of(path, "..", "..", "README.md"));
 
             return lines.stream().filter(line -> README_PATTERN.matcher(line).find())
                     .collect(Collectors.toMap(this::lineToNutName, Function.identity()));
-        } catch (final IOException e) {
-            fail("Could not read README.md from: " + path);
+        } catch (final IOException | URISyntaxException e) {
+            fail("Could not read README.md");
             return null;
         }
     }
 
     private List<String> readThingsXml(final Pattern pattern, final String filename) {
-        final String path = getClass().getProtectionDomain().getClassLoader().getResource(".").getFile()
-                + "../../src/main/resources/OH-INF/thing";
         try {
-            final List<String> lines = Files.readAllLines(Paths.get(path, filename));
+            final String path = Path.of(getClass().getProtectionDomain().getClassLoader().getResource(".").toURI())
+                    .toString();
+            final List<String> lines = Files
+                    .readAllLines(Path.of(path, "..", "..", "src", "main", "resources", "OH-INF", "thing", filename));
             return lines.stream().filter(line -> pattern.matcher(line).find()).map(String::trim).sorted()
                     .collect(Collectors.toList());
-        } catch (final IOException e) {
-            fail("Could not read things xml from: " + path);
+        } catch (final IOException | URISyntaxException e) {
+            fail("Could not read things xml");
             return null;
         }
     }

